modules/04 Advanced Javascript/Assignments/4/AdvancedJS4.js

```diff
@@ -0,0 +1,25 @@
+Array.prototype.min = function(){
+    var min = Infinity;
```

Comment:
why infinity on "minimum"?

Reply:
Because I need a large number such that whatever big numbers there is in the array my number will always "wins"